doc/source/admin/framework_dependencies.rst

```diff
@@ -3,85 +3,72 @@
-
-2. Unset the ``$PYTHONPATH`` environment variable (if set) as this can
-   interfere with installing `Galaxy pip`_ and dependencies.
+1. Create a Python virtualenv_ in the directory ``.venv``.
 
-3. Replace that virtualenv's pip with `Galaxy pip`_.
+2. Unset the ``$PYTHONPATH`` environment variable (if set) as this can interfere with installing dependencies.
 
-4. If applicable, create a ``binary-compatibility.cfg`` (see the `Galaxy pip
-   and wheel`_ section for an explanation of this file).
+3. Download and install packages from the Galaxy project wheel server, wheels.galaxyproject.org_, as well as the `Python
+   Package Index`_ (aka PyPI) , using pip_.
 
-5. Download and install wheels from the Galaxy project wheel server,
-   `wheels.galaxyproject.org`_, using pip.
+4. Start Galaxy using ``.venv/bin/python``.
 
-6. Start Galaxy using ``.venv/bin/python``.
-
-.. _virtualenv: https://virtualenv.readthedocs.org/
 .. _wheels.galaxyproject.org: https://wheels.galaxyproject.org/
+.. _Python Package Index: https://pypi.org
 
 Options
 -------
 
 A variety of options to ``run.sh`` are available to control the above behavior:
 
-- ``--skip-venv``: Do not create or use a virtualenv, and do not replace pip
-  with Galaxy pip.
+- ``--skip-venv``: Do not create or use a virtualenv.
 - ``--skip-wheels``: Do not install wheels.
-- ``--no-create-venv``: Do not create a virtualenv, but use one if it exists at
-  ``.venv`` or if ``$VIRTUAL_ENV`` is set (this variable is set by virtualenv's
-  ``activate``)
-- ``--replace-pip/--no-replace-pip``: Do/do not replace pip with Galaxy pip.
+- ``--no-create-venv``: Do not create a virtualenv, but use one if it exists at ``.venv`` or if ``$VIRTUAL_ENV`` is set
+  (this variable is set by virtualenv's ``activate``).
+- ``--replace-pip/--no-replace-pip``: Do/do not upgrade pip if necessary.
 
 Managing dependencies manually
 ------------------------------
 
 Create a virtualenv
 ^^^^^^^^^^^^^^^^^^^
 
-Using a `virtualenv`_ in ``.venv`` under the Galaxy source tree is not
-required. More complicated Galaxy setups may choose to use a virtualenv
-external to the Galaxy source tree, which can be done either by not using
-``run.sh`` directly (an example of this can be found under the `Supervisor`_
-section) or using the ``--no-create-venv`` option, explained in the `Options`_
-section. It is also possible to force Galaxy to start without a virtualenv at
-all, but you should not do this unless you know what you're doing.
+Using a `virtualenv`_ in ``.venv`` under the Galaxy source tree is not required. More complicated Galaxy setups may
+choose to use a virtualenv external to the Galaxy source tree, which can be done either by not using ``run.sh`` directly
+(an example of this can be found under the `Scaling and Load Balancing` documentation) or using the ``--no-create-venv``
+option, explained in the `Options` section. It is also possible to force Galaxy to start without a virtualenv at all,
```

Comment:
Underscore missing after \`Options\` ?

Reply:
I guess I was thinking of Sphinx syntax for cross refs which don't carry the following underscore?: 

```
:ref:`cross ref`
```

Reply:
Yep, that's probably what you want to use, see https://stackoverflow.com/questions/15394347/adding-a-cross-reference-to-a-subheading-or-anchor-in-another-page